searchtool/package.json

```diff
@@ -11,7 +11,7 @@
     }
   ],
   "name": "officeactions",
-  "description": "USPTO Data Search Framework",
+  "description": "officeactions",
```

Comment:
Not sure why this is being changed, since it's going to cover more than office actions now. If anything I'd say "name" should be "uspto_search" and description should be a bit more verbose.